sarah/S3/S3.m

```diff
@@ -0,0 +1,107 @@
+Off[General::spell]
+
+Model`Name = "S3";
+Model`NameLaTeX ="S3 Leptoquark";
+Model`Authors = "Daniele Rizzo, Abhishek Chikkaballi Ramalingegowda and Wojciech Kotlarski";
+Model`Date = "2023-04-17";
+
+(*-------------------------------------------*)
+(*Starting comment from SM implementation*)
```

Comment:
Are these comments correct? (I thought it is a new model file that you've wrote. :) )

Reply:
Removing them would be good, in my opinion, because I think the SM history does not apply here / does not make sense here.

Reply:
Tweaked in 9cf2b46a8b9461d34ba5c96c0f2b661e9e0ec5e3.